docs/node/run-your-node/rofl-node.mdx

```diff
@@ -0,0 +1,332 @@
+
+```shell
+oasis rofl provider create
+```
+
+The account signing the transaction is now registered as a ROFL provider
+on-chain. In our case, the built-in `test:erin` account which we used for
+signing has address `oasis1qqcd0qyda6gtwdrfcqawv3s8cr2kupzw9v967au6`.
+
+:::info
+
+Registering a new ROFL provider requires depositing
+[100 tokens][stake-requirements] which are returned to you, when you deregister
+it.
+
+:::
+
+[stake-requirements]: ./prerequisites/stake-requirements.md
+
+[Oasis CLI]: https://github.com/oasisprotocol/cli/blob/master/docs/README.md
+
+#### Configure your ROFL node for the marketplace {#configure-rofl-node-marketplace}
+
+1. Download the [latest release of the Scheduler ROFL app][rofl-scheduler]
+   and save it to your ROFL node, for example `/node/rofls/`. This app will
+   listen to ROFL hosting requests, configure any incoming ROFLs and spin them
+   up. It will also listen to ROFL admin request for stopping or restarting the
+   ROFL machine.
+2. Add the Scheduler ROFL app to your Oasis node's `config.yml` inside the
+   `runtime.paths` and configure the Scheduler specifics such as the provider
+   address, acceptable offers and capacities on this node:
+
+    ```yaml title="config.yml"
+    runtime:
+      sgx:
+        loader: /node/bin/oasis-core-runtime-loader
+      paths:
+        - /node/rofls/rofl-scheduler.testnet.orc
+      runtimes:
+        # Sapphire Testnet RONL with SGX
+        - id: "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c"
+          config:
+            allowed_queries:
+              - all_expensive: true
+          components:
+           - id: rofl.rofl1qrqw99h0f7az3hwt2cl7yeew3wtz0fxunu7luyfg # ROFL scheduler app ID, should not change
+             permissions:
+             - bundle_add
+             - bundle_remove
+             - volume_add
+             - volume_remove
+             config:
+               rofl_scheduler:
+                 provider_address: oasis1qqcd0qyda6gtwdrfcqawv3s8cr2kupzw9v967au6 # Your provider address
+                 offers:
+                   - small # Your offer name(s)
+                 capacity:
+                   instances: 24
+                   memory: 65536
+                   cpus: 24
+                   storage: 549755813888
+    ```
+3. Restart your Oasis node. After a while, your ROFL node will be ready to
+   accept ROFLs. ROFL app developers can now simply deploy their ROFL to your
+   node by providing the `--provider <your address>` to the [`oasis rofl deploy`]
+   command.
+
+   ```shell
+   oasis rofl deploy --provider oasis1qqcd0qyda6gtwdrfcqawv3s8cr2kupzw9v967au6
+   ```
+
+:::tip Multiple ROFL nodes
+
+If you configured multiple ROFL nodes for a single provider, the machine
+instantiated to execute the ROFL app will be arbitrarily picked depending on
+which ROFL node register transaction appears first one on the chain.
+
+:::
+
+#### Multiple ROFL replicas on a single node {#rofl-app-id-remap}
```

Comment:
This seems like an internal detail, not sure if we should mention it in the documentation? Especially once log view support is live.

Reply:
I shortened this and put it inside a tip admonition. I think it's important for ROFL node operators to know there can be multiple ROFL instances of the same app running on their machine.